docs/root/intro/arch_overview/upstream/load_balancing_policies.rst

```diff
@@ -1,20 +1,21 @@
 Load balancing policies
 =======================
 
-Extendable load balancing policies could be configured for each cluster by
-:ref:`load balancer policy <envoy_v3_api_field_config.cluster.v3.Cluster.load_balancing_policy>`.
-And the developer can implement a custom load balancing policy and configured it.
+Extendable load balancing policies can be
+:ref:`configured <envoy_v3_api_field_config.cluster.v3.Cluster.load_balancing_policy>` separately for each cluster, also by calling
+:ref:`APIs <envoy_v3_api_config_load_balancer_policies>`.
 
+Developers can implement custom, configurable policies in C++.
 
-See built-in load balancing policies by :ref:`APIs <envoy_v3_api_config_load_balancer_policies>`.
+.. note::
+  In the past, Envoy used an
+  :ref:`enum <envoy_v3_api_field_config.cluster.v3.Cluster.lb_policy>`
+  to specify load balancing policies. This ``enum`` is still supported for
+  backward compatibility, but deprecated;
+  ref:`extendable load balancing policies <envoy_v3_api_config_load_balancer_policies>`
```

Comment:
```suggestion
  :ref:`extendable load balancing policies <envoy_v3_api_config_load_balancer_policies>`
```

apologies - spotted a rendering issue at the last minute

Reply:
also - while you are there - could i suggest we put this to a separate line and replace `;` with `.` after "...deprecated."

i think this is the important bit

you will need a single blank line to render it correctly in rst